plugin/src/main/scala/io/github/davidgregory084/ScalacOptions.scala

```diff
@@ -653,6 +653,39 @@ private[davidgregory084] trait ScalacOptions {
     warnError
   )
 
+  /** Verbose options (-V)
+    */
+  def verboseOption(name: String, isSupported: ScalaVersion => Boolean = _ => true) =
+    ScalacOption(s"-V$name", isSupported)
+
+  /** Print dependent missing implicits.
+    */
+  val verboseImplicits =
+    verboseOption("implicits", _.isBetween(V2_13_0, V3_0_0))
```

Comment:
I found that this was added as an alias of `-Xlog-implicits` for Scala 2.13.0 in [#7908](https://github.com/scala/scala/pull/7908).

Reply:
Just want to check @sideeffffect whether there was a reason you only wanted to enable this above 2.13.6? Did the behaviour change a lot? I seem to remember `-Xlog-implicits` used to produce a ton of output.